packages/patternfly-4/react-core/src/styles/sizes.js

```diff
@@ -4,9 +4,9 @@ export const BaseSizes = {
   md: 'md',
   lg: 'lg',
   xl: 'xl',
-  '2xl': '2xl',
-  '3xl': '3xl',
-  '4xl': '4xl'
+  xl2: 'xl2',
+  xl3: 'xl3',
+  xl4: 'xl4'
```

Comment:
I was just looking for TitleSize.2xl to be added. Why are we changing all the size strings? 

Note that this would be a breaking change and does not appear to solve my original issue with TitleSize.

Reply:
@dlabrecq Sorry for not clarifying -- JavaScript prohibits variable names from starting with numbers, which is why 2xl must be accessed as a string: TitleSize['2xl'].

@tlabaj marked this as a breaking change issue because there isn't a quick fix that would allow for TitleSize.2xl without renaming the sizes

Reply:
@dlabrecq do you prefer this over using bracket notation `Size['2xl']` to access the variable?
other breaking options we could consider:
```
1. Size.xl2 (this PR)
2. Size.xxl
3. Size._2xl
4. Size.$2xl
```

Reply:
To be honest, I thought Size['2xl'] was a bug. If that's how it's intended to be used, then we can just add a comment and close the issue? 

Anything would be more readable, tho. Although, Size._2xl looks like the naming convention for a private variable. My vote would be for #1, what's in this PR now.

Whatever is chosen, TitleSize just needs to be in sync.